plugins/chunter-impl/src/components/UserSpaceInfo.svelte

```diff
@@ -55,7 +55,7 @@
     <div class="caption">
       <div class="title" class:header={headerSize} class:item={!headerSize}>
         {accounts
-          .map((u) => u.name)
+          .map((u) => (u.name.trim().length > 0 ? u.name : u.email))
```

Comment:
`.map((u) => (u.name.trim().length > 0 ? u.name.trim() : u.email))` ?

Reply:
If name is not specified, email will be used.

Reply:
@haiodo No. please look at `? u.name.trim() : `